.github/workflows/migration.yml

```diff
@@ -0,0 +1,55 @@
+name: Database Migration Test
+on:
+  workflow_call:
+jobs:
+    test:
+      name: MySQL -> Postgres Migration
+      runs-on: ubuntu-22.04
+      env:
+          COMPOSE_PROJECT_NAME: ghactions
+          BUILD_IMAGE: mattermost/mattermost-build-server:20230118_golang-1.19.5
+      defaults:
+        run:
+          working-directory: server
+      steps:
+      - name: Checkout mattermost project
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Run docker compose
+        run: |
+          cd build
+          docker-compose --ansi never run --rm start_dependencies
+          cat ../tests/test-data.ldif | docker-compose --ansi never exec -T openldap bash -c 'ldapadd -x -D "cn=admin,dc=mm,dc=test,dc=com" -w mostest';
```

Comment:
Do we really need ldap data?

Reply:
Hmm, I think not for the test-data, it was copied form another workflow. I'll remove.